Backport a97271e3b5d5a08fc503a11cd3e253974fb77ce6

varada1110 opened the pull request:
Hi all,

This pull request contains a backport of commit [a97271e3](https://github.com/openjdk/jdk/commit/a97271e3b5d5a08fc503a11cd3e253974fb77ce6) from the [openjdk/jdk](https://git.openjdk.org/jdk) repository.

The commit being backported was authored by Weijun Wang on 6 Mar 2023 and was reviewed by Sean Mullan.

Thanks!

GoeLin commented:
Hi Varada, 
please first get a review, then label for approval.
Please edit the description of the PR and tell why this is not clean.
I'm not sure this is a good backport. Please tell in the approval comment in JBS why you 
want to backport this and give a risk assessment.

varada1110 commented:
Hi @seanjmullan , 
Could you please review this PR?

phohensee commented:
You might ask @weijun, who fixed the original issue.

GoeLin commented:
Hi @varada1110 
please state in the PR comment why this is not clean.
please give a reason why you want this in 17 (besides telling this fixes the error it is supposed to fix...).  
GHA and the related tests are a minimum.  Did you do any other testing?

varada1110 commented:
> Hi @varada1110 please state in the PR comment why this is not clean. please give a reason why you want this in 17 (besides telling this fixes the error it is supposed to fix...). GHA and the related tests are a minimum. Did you do any other testing? (I succesfully ran this through SAPs testing)

Hi @GoeLin ,
I have updated the PR description and added why PR Is not clean and reason for backporting it to jdk17.
Thank you for testing from your end. I have only done jtreg testing. Do I need to perform any other testing, please let me know.

GoeLin commented:
Hi @varada1110 
I see this did not get backported in the end. 
I think this is because you did not add the label again.
I ran into this backporting https://bugs.openjdk.org/browse/JDK-8367008: Algorithm identifiers for HmacSHA* should always have NULL as params
Could you please revive this? 
I.e., open it again, merge head so that it runs through the tests again, and label for approval?
Best, Goetz.

varada1110 commented:
/open

varada1110 commented:
/open

GoeLin commented:
Thank you, @varada1110 
Now you can add label jdk17u-fix-request to the JBS issue.

varada1110 commented:
Thanks @GoeLin , I have added the label: jdk17u-fix-request

GoeLin commented:
approved.

varada1110 commented:
/integrate

GoeLin commented:
/sponsor

GoeLin commented:
Thanks for driving this, @varada1110
